cmd/harmony/main.go

```diff
@@ -290,6 +290,10 @@ func setupNodeAndRun(hc harmonyconfig.HarmonyConfig) {
 	// Update ethereum compatible chain ids
 	params.UpdateEthChainIDByShard(nodeConfig.ShardID)
 
+	if hc.Feature != nil && hc.Feature.EnablePruneBeaconChain {
+		nodeConfig.Feature.EnablePruneBeaconChain = hc.Feature.EnablePruneBeaconChain
```

Comment:
Please make sure the beacon chain pruning config only works on non shard 0 validators.

Reply:
This is already limited, see:

https://github.com/harmony-one/harmony/pull/3908/files#diff-375d57e386f20eaa5f09f02bb9d28bfc48ac3dca18d0325f59492208219e5618R159